src/mcp_gateway/__main__.py

```
@@ -0,0 +1,17 @@
+import sys
+
+from mcp_gateway.audit import audit_logger
+from mcp_gateway.server import run_gateway
+
+
+def main() -> None:
+    """Entry point for the chronos-mcp-gateway."""
+    try:
+        run_gateway()
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)